trailblazer/server/api.py

```diff
@@ -108,9 +107,8 @@ def me():
 @blueprint.route("/aggregate/jobs")
 def aggregate_jobs():
     """Return stats about failed jobs."""
-    time_window: datetime = get_date_number_of_days_ago(
-        number_of_days_ago=int(request.args.get("days_back", ONE_MONTH_IN_DAYS))
-    )
+    number_of_days_ago: int = int(request.args.get("days_back", ONE_MONTH_IN_DAYS))
```

Comment:
Does this count as an instantiation of an `int` in that we can remove the type hint 🤔 Either way, you can disregard this comment